game.rb

```diff
@@ -24,21 +23,28 @@ def player_one_pick_word
   end
 
   def player_two_guess_word word
-    counter = 0
+    guess_counter = 0
     end_of_game = false
+    @io_handler.print(@game_view.body_array)
     while ! end_of_game
```

Comment:
You can decide on the best way to do this but this method is definitely too hard to follow.